src/main/java/org/takes/facets/fork/RsFork.java

```diff
@@ -62,17 +62,10 @@ public RsFork(final Request req, final Fork... list) {
      */
     public RsFork(final Request req, final Iterable<Fork> list) {
         super(
-            new Response() {
-                @Override
-                public Iterable<String> head() throws IOException {
-                    return RsFork.pick(req, list).head();
-                }
-
-                @Override
-                public InputStream body() throws IOException {
-                    return RsFork.pick(req, list).body();
-                }
-            }
+            new ResponseOf(
+                () -> RsFork.pick(req, list).head(),
+                () -> RsFork.pick(req, list).body()
```

Comment:
@fanifieiev same above.

Reply:
@fabriciofx According to your above clarification, this piece of code should be left as it is, right?

Reply:
@fanifieiev yes.